documentation/docs/developers/strategies/simple-trade.md

```diff
@@ -0,0 +1,12 @@
+# Simple Trade
+
+## Overview
+
+Simple trade strategy expands upon Perform Trade strategy. It has a few additional features such as time constraints and order specific loggings.
```

Comment:
It's better to describe the differences in terms of what the tutorial reader can learn from the strategy's code. The primary purpose of the tutorial is to teach a reader new tricks he or she can do inside a strategy, or a new mechanism with the strategy code that can be used - in this case, using the `c_tick()` mechanism to add a time delay before the trade.

Reply:
Another thing I'd suggest to add is a simple flow chart, like the perform trade documentation. Visual aids are useful for developers to grasp new concepts quickly.